thoth/messaging/update_is_downloadable_ppv.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+# thoth-messaging
+# Copyright(C) 2020 Kevin Postlethwait
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""This is Thoth Messaging module for UpdateIsDownloadableMessage."""
+
+from .message_base import MessageBase, BaseMessageContents
+
+
+class UpdateIsDownloadableMessage(MessageBase):
+    """Class used for updating python package version database flag."""
+
+    topic_name = "thoth.security-indicator.update-is-downloadable"
+    _message_version = 1  # update on schema change
+
+    class MessageContents(BaseMessageContents, serializer="json"):  # type: ignore
+        """Class used to represent a contents of a update-is-downloadable message."""
+
+        package_name: str
+        package_version: str
+        index_url: str
+        value: bool
```

Comment:
what goes into value? if this message is sent we know we have to set the flag to False in storages

Reply:
I would like the ability for the message to be reversible.  It is possible to upload a source distro for a package after the fact and we may want to watch for that eventually